src/app/organizer/Columns.tsx

```diff
@@ -576,22 +605,12 @@ function PerksCell({
   if (!item.sockets) {
     return null;
   }
-
-  let sockets = item.sockets.categories.flatMap((c) =>
-    getSocketsByIndexes(item.sockets!, c.socketIndexes).filter(
-      (s) =>
-        !isKillTrackerSocket(s) &&
-        !isEmptyArmorModSocket(s) &&
-        s.plugged?.plugDef.displayProperties.name && // ignore empty sockets and unnamed plugs
-        (s.plugged.plugDef.collectibleHash || // collectibleHash catches shaders and most mods
-          isUsedArmorModSocket(s) || // but we catch additional mods missing collectibleHash (arrivals)
-          (s.isPerk &&
-            (item.isExotic || // ignore archetype if it's not exotic
-              !s.plugged.plugDef.itemCategoryHashes?.includes(
-                ItemCategoryHashes.WeaponModsIntrinsic
-              ))))
-    )
+  const perks = item.sockets.categories.filter(
+    (c) =>
+      c.category.displayProperties.name !== 'WEAPON MODS' &&
+      c.category.displayProperties.name !== 'WEAPON COSMETICS'
```

Comment:
We can't rely on displayProperties because these are different between different languages. Try `SocketCategoryHashes`.

Reply:
Ah I see, so those would also be completely unique to each category?